test/integration/auth/mongodb_aws.test.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-const expect = require('chai').expect;
+const { expect } = require('chai');
 
 describe('MONGODB-AWS', function () {
   before(function () {
```

Comment:
A slight enhancement while we're here, now that these will show up in the other variants can we add a skipReason to this hook?

Reply:
Any particular suggestion for the skip reason?

Reply:
cus of this: `MONGODB_URI.indexOf('MONGODB-AWS')` it could be `test requires MONGODB_URI to contain 'MONGODB-AWS' auth mechanism` I don't think you need to attach the value of MONGODB_URI cus its logged elsewhere

Reply:
Will include in the next refactor